ci/linux_apisix_master_luarocks_runner.sh

```diff
@@ -39,7 +39,7 @@ script() {
     cp -r ../utils ./
 
     # install APISIX by luarocks
-    luarocks install $APISIX_MAIN > build.log 2>&1 || (cat build.log && exit 1)
+    luarocks install $APISIX_LDAP_DEPRECATION > build.log 2>&1 || (cat build.log && exit 1)
```

Comment:
ditto.

Reply:
ack, will be replacing the reference of my own branch to mainline, once we are ready to merge this PR